XiEditor/Main.storyboard

```diff
@@ -1015,6 +1014,20 @@ Gw
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <menu key="submenu" title="View" id="SBh-Xs-jHf">
                                     <items>
+                                        <menuItem title="Theme" id="SUL-oD-daT">
```

Comment:
Okay so this is totally my fault (going back and rereading my comment about moving the Theme menu):

I think this change might make sense, but I'm not sure `Theme` should go under `View` (it might belong with preferences under the main 'XiEditor' menu?), and in any case it feels like a change that is pretty firmly unrelated to this PR, so it would make more sense to do it separately. Sorry for the confusion!

Reply:
Yeah, I thought about it being unrelated to the PR, but figured that if I mentioned that in the PR description it would be okay. But sure, let's do this separately, that would be better.